_docs/quick-start.md

```diff
@@ -38,11 +41,20 @@ We recommend that you write your TypeScript in [VS Code](https://code.visualstud
 
 1. Create a new folder and name it whatever you'd like. For example: `MyCoolProject`
 
-2. Run `npm init -y` inside of your folder. This will create your `package.json` file.
+2. Run `rbxtsc --init game` inside your folder. This will create the required files to get you started:
+
+<p align="center"><img src=https://i.imgur.com/Rlrw534.png></p>
+
+It's recommended that you don't run `rbxtsc --init` in a non-empty folder, as in the worst case it might lead to loss of data!
+{:.alert}
+
+3. After generation, your directory should look like this:
 
-3. Create a folder called `src` to contain your TypeScript files.
+<p align="center"><img src=https://i.imgur.com/hNPk82S.png></p>
```

Comment:
.. you should have a `src` folder??

Reply:
For some reason it did not make one automatically, which is strange given you explicitly told it would.

Doesn't seem to be platform specific since that I get the same behavior in Ubuntu WSL:

![image](https://user-images.githubusercontent.com/45321798/61891630-f9a9e580-af01-11e9-9499-ce4254590cb4.png)

Reply:
That's super odd. Let me investigate and see if we can get that fixed before telling users to use it.

Reply:
Fixed.